### ISSUE TITLE
[DEV Question] List of URLs

### ISSUE THREAD

**Issue by Typhonragewind** [webcomics/dosage#272]
In multi image comic classes, is there any way to retrieve the list of gathered image urls (something like self.image_urls in comic.py) from the "self" variable at the custom namer level?

**Comment by TobiX** [comment]
Not really... But there are already some modules which work around this limitation, for example:
https://github.com/webcomics/dosage/blob/52d9f38793078e1c86236bfb463fc85d8fe1af2a/dosagelib/plugins/tapas.py#L40-L43
So it would probably be a good idea to extend the standard API with these properties...

**Comment by Typhonragewind** [comment]
That looks exactly like what I needed, thanks!

Do you know of other examples I could refer to? I doesn't seem to work for me, so I might be doing something wrong and want to compare to others
